utils/helpers/recommendations/getRecommendationsData.ts

```diff
@@ -0,0 +1,20 @@
+'use server';
+import { getAllUsers } from '@/actions/getUsers';
+import { getAllFollowees } from '@/utils/helpers/followers/getFollowees';
+import { IMumbleUser } from '@/utils/interfaces/mumbleUsers.interface';
+
+// in the real world the api should deliver  recommendations based on user preferences/behaviour/marketing
+// we just get all the users and the user followees. In a real world scenario we would set a batch number of recommendations
```

Comment:
Would it be a big effort to implement the batch number of recommendations? If not, we could implement it as it is a real world scenario.